jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/handler/ProbeHandlerTest.java

```diff
@@ -149,14 +143,14 @@ public void getTCPProbeWithoutURLTest() {
 
         probe = probeHandler.getProbe(probeConfig);
         //assertion
-        assertNotNull(probe);
-        assertNull(probe.getHttpGet());
-        assertNotNull(probe.getTcpSocket());
-        assertEquals(80,probe.getTcpSocket().getPort().getIntVal().intValue());
-        assertNull(probe.getTcpSocket().getHost());
-        assertNull(probe.getExec());
-        assertEquals(5,probe.getInitialDelaySeconds().intValue());
-        assertEquals(5,probe.getTimeoutSeconds().intValue());
+        assertThat(probe).isNotNull();
+        assertThat(probe.getHttpGet()).isNull();
+        assertThat(probe.getTcpSocket()).isNotNull();
+        assertThat(probe.getTcpSocket().getPort().getIntVal().intValue()).isEqualTo(80);
+        assertThat(probe.getTcpSocket().getHost()).isNull();
+        assertThat(probe.getExec()).isNull();
+        assertThat(probe.getInitialDelaySeconds().intValue()).isEqualTo(5);
+        assertThat(probe.getTimeoutSeconds().intValue()).isEqualTo(5);
```

Comment:
Please chain assertions here. Try to do assertion using one `assertThat` statement like done in other places.